akka-persistence-typed/src/test/java/jdocs/akka/persistence/typed/PersistentFsmToTypedMigrationCompileOnlyTest.java

```diff
@@ -97,7 +98,7 @@ public String manifest(DomainEvent event) {
 
     @Override
     public EventSeq<DomainEvent> fromJournal(Object event, String manifest) {
-      if (event instanceof StateChangeEvent) {
+      if (event instanceof akka.persistence.fsm.PersistentFSM.StateChangeEvent) {
```

Comment:
Is this consistent with the changed import?

Reply:
Yes: the changed import covered the `WebStoreCustomerFSM` classes from akka-persistence/src/test, while StateChangeEvent is from akka-persistence/src/main